src/instana/span/registered_span.py

```diff
@@ -139,8 +139,8 @@ def _populate_local_span_data(self, span) -> None:
         if span.name == "render":
             self.data["render"]["name"] = span.attributes.pop("name", None)
             self.data["render"]["type"] = span.attributes.pop("type", None)
-            self.data["event"]["message"] = span.attributes.pop("message", None)
-            self.data["event"]["parameters"] = span.attributes.pop("parameters", None)
+            self.data["log"]["message"] = span.attributes.pop("message", None)
+            self.data["log"]["parameters"] = span.attributes.pop("parameters", None)
```

Comment:
Why are you assuming it's not an event but a log?

Reply:
The backend expects the entry to be `span.data["log"]` so when we supply it as `span.data["event"]` it doesn’t recognise the same.